docs/docs-content/vm-management/create-manage-vm/deploy-vm-from-template.md

```diff
@@ -55,6 +55,15 @@ These steps will help guide you to deploy a VM from an out-of-the-box VM templat
 
 8. Give the VM a name and specify memory and CPUs.
 
+   :::info
+
+   You can change the memory and CPU values after the VM is deployed. By default, the Hotplug feature is enabled for
```

Comment:
```suggestion
   You can change the memory and CPU values after the VM is deployed. By default, the hotplug feature is enabled for
```

I think we don't capitalize for most instances of hotplug in our docs and it's not capitalized in the kubevirt docs either,

Reply:
Nice catch